src/fdk_rdf_parser/classes/cpsvno_service.py

```diff
@@ -54,3 +58,6 @@ class Service:
     subject: Optional[List[str]] = None
     homepage: Optional[List[str]] = None
     dctType: Optional[List[ReferenceDataCode]] = None
+    thematicareas: Optional[List[EuDataTheme]] = None
```

Comment:
```suggestion
    thematicAreas: Optional[List[EuDataTheme]] = None
```

Reply:
Så nå har vi thematicAreas, som er alle tema, losThemes, som er de vi klarer å matche mot los og euDataThemes som er de vi klarer å matche mot eu?

Reply:
kewl, fungerer bedre enn hva vi gjør for datasett :+1:

Reply:
Men siden thematicAreas aldri blir får ekstra data så er det bedre om du ikke bruker classen EuDataTheme

Reply:
> Så nå har vi thematicAreas, som er alle tema, losThemes, som er de vi klarer å matche mot los og euDataThemes som er de vi klarer å matche mot eu?

Stemmer! Tar gjerne mot forbedringsforslag på strukturen, prøvde å følge flyten i Datasett og gjenbruke det i Service. Blir jo en del dataduplisering her, hver entry fins to ganger.

Kan alternativt droppe `thematicAreas` og kun sette `losThemes` og `euDataThemes`. Hva tenker du?

> Men siden thematicAreas aldri blir får ekstra data så er det bedre om du ikke bruker classen EuDataTheme

Mener du å lage en separat `Theme`-klasse for thematicAreas?

Reply:
> Mener du å lage en separat Theme-klasse for thematicAreas?
Joa går det, men hvis eneste dataen som ligger der er en uri, hvorfor lage en klasse?

> Kan alternativt droppe thematicAreas og kun sette losThemes og euDataThemes. Hva tenker du?
Dropping av thematicAreas er et fint alternativ. Men igjen, hva skjer da med ikke matchende uri'er?

Reply:
Ja, bedre med streng/URI!
Ok, da beholder vi ikke-matchende uri-er.

Begge deler krever refaktorering av split_themes og Dataklasse-parsingen, så vidt jeg kan se. Lager ny commit på det.

Reply:
Vi kan dog prøve å hente den ut som SkosConcept, siden det er det som er faktisk type for feltet

Da vil vi klare å få med data som er lagt til på grafen om tema som ikke er en del av data-theme eller los

Reply:
men enn så lenge skal vi bare vise de vi klarer å matche som data-theme eller los, ingenting annet i designet

Reply:
legg det til som en liste av streng/URI, trenger ikke å lage merarbeid uten grunn. Vi får tenke mer på det om det blir aktuelt å vise mer enn data-theme og los i portalen

Reply:
Kan lage issue på det også.

Reply:
ok, men det er langt fra sikkert at vi faktisk skal gjøre noe med det. Forklar situasjonen og legg det i new, vi kan flytte det til backlog/ready hvis det blir klart at vi må gjøre en endring. Blir det korrekt måte å bruke utviklingsbrettet? Kanskje dobbeltsjekk med kjersti :)